packages/npm2gpr/index.js

```diff
@@ -0,0 +1,21 @@
+const fetch = require("node-fetch");
+const url1 = "https://registry.npmjs.org/-/v1/search?text=@creately&from=0";
```

Comment:
The last part of the url must be generated dynamically. The values depend on the result set size.

```ts
function getUrl( from ) {
  return `https://registry.npmjs.org/-/v1/search?text=@creately&from=${ from }`;
}
```

Reply:
done